### PR TITLE
add server mode option

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**

this option allows runs the reports and do not remove the images.
It will be helpful to generate them faster and reduce the time required to download all layers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ $ make install
 
 ### Ensure that you have access to pull the images
 
-You need first use docker login to have access to the images. 
+You need first use docker login to have access to the images.
```
Comment (bentito) [inline README.md:47]:
```suggestion
You must run `docker login` to have access to the images.
```

File: README.md [hunk 2]
```diff
@@ -98,6 +98,13 @@ See that you can use the `--filter` --flag to filter the results by the package
 audit-tool index [bundles|packages|channels] --index-image=registry.redhat.io/redhat/redhat-operator-index:v4.5 --filter="mypackagename"
 ```
 
+### Option to run in dedicated environments
+
+Use the flag `--server-mode` to generate the reports in dedicated environments. By using this flag option the images
+which are downloaded will not be removed which will allow the reports to be generate faster after the first execution.
```
Comment (bentito) [inline README.md:104]:
```suggestion
which are downloaded will not be removed, allowing the reports to be generated faster after the first execution. 
```

File: README.md [hunk 3]
```diff
@@ -98,6 +98,13 @@ See that you can use the `--filter` --flag to filter the results by the package
 audit-tool index [bundles|packages|channels] --index-image=registry.redhat.io/redhat/redhat-operator-index:v4.5 --filter="mypackagename"
 ```
 
+### Option to run in dedicated environments
+
+Use the flag `--server-mode` to generate the reports in dedicated environments. By using this flag option the images
+which are downloaded will not be removed which will allow the reports to be generate faster after the first execution. 
+
+Also, ensure that you have enough space to store all images. Note that the default behavior is to remove them, when this option is not used.
```
Comment (bentito) [inline README.md:106]: Might want to give a size guess so admins know how much space to allocate.

File: cmd/index/bundles/command.go [hunk 4]
```diff
@@ -82,6 +82,9 @@ func NewCmd() *cobra.Command {
 	cmd.Flags().StringVar(&flags.LabelValue, "label-value", "",
 		"filter by bundles which has index images where contains *label=label-value*. "+
 			"This option can only be used with the --label flag.")
+	cmd.Flags().BoolVar(&flags.ServerMode, "server-mode", false,
+		"if set, the image which are downloaded will not be removed. This flag should be used on dedicated "+
```
Comment (bentito) [inline cmd/index/bundles/command.go:86]:
```suggestion
		"if set, the images which are downloaded will not be removed. This flag should be used on dedicated "+
```

File: cmd/index/bundles/command.go [hunk 5]
```diff
@@ -82,6 +82,9 @@ func NewCmd() *cobra.Command {
...
+			"environments and to reduce the cost to generate the reports periodically")
```
Comment (bentito) [inline cmd/index/bundles/command.go:87]:
```suggestion
			"environments to reduce the cost to generate the reports periodically")
```

File: cmd/index/packages/command.go [hunk 6]
```diff
@@ -82,6 +82,9 @@ func NewCmd() *cobra.Command {
 	cmd.Flags().StringVar(&flags.LabelValue, "label-value", "",
 		"filter by packages which has bundles with index images where contains *label=label-value*. "+
 			"This option can only be used with the --label flag.")
+	cmd.Flags().BoolVar(&flags.ServerMode, "server-mode", false,
+		"if set, the image which are downloaded will not be removed. This flag should be used on dedicated "+
```
Comment (bentito) [inline cmd/index/packages/command.go:86]:
```suggestion
		"if set, the images which are downloaded will not be removed. This flag should be used on dedicated "+
```

File: cmd/index/packages/command.go [hunk 7]
```diff
@@ -82,6 +82,9 @@ func NewCmd() *cobra.Command {
...
+			"environments and to reduce the cost to generate the reports periodically")
```
Comment (bentito) [inline cmd/index/packages/command.go:87]:
```suggestion
			"environments to reduce the cost to generate the reports periodically")
```

File: README.md (hunk 3 above)
Comment (camilamacedo86) [reply README.md:106]: Then that is up to the images that will be used.

**Review by jmccormick2001 — APPROVED** [review]

**Pull request merged**